package.json

```diff
@@ -73,6 +73,7 @@
     "eslint-config-prettier": "^6.11.0",
```

Comment:
just checking which npm version you have for the package-lock changes

Reply:
I'm on `6.14.13` now, these changes are going back from `lockFileVersion: 2` to `lockFileVersion: 1`